crmsh/bootstrap.py

```diff
@@ -2337,13 +2343,16 @@ def bootstrap_join(context):
         if not utils.service_is_active("pacemaker.service", cluster_node):
             error("Cluster is inactive on {}".format(cluster_node))
 
-        lock_inst = join_lock.JoinLock(cluster_node)
-        with lock_inst.lock():
-            setup_passwordless_with_other_nodes(cluster_node)
-            join_remote_auth(cluster_node)
-            join_csync2(cluster_node)
-            join_ssh_merge(cluster_node)
-            join_cluster(cluster_node)
+        lock_inst = lock.RemoteLock(cluster_node)
+        try:
+            with lock_inst.lock():
+                setup_passwordless_with_other_nodes(cluster_node)
+                join_remote_auth(cluster_node)
+                join_csync2(cluster_node)
+                join_ssh_merge(cluster_node)
+                join_cluster(cluster_node)
+        except (lock.SSHError, lock.ClaimLockError) as err:
```

Comment:
I see that `ValueError` is raised in `RemoteLock`. I guess we need to capture it here too